.github/workflows/autoupdate-sdk.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
         run:  echo "VERSION=$(curl -sSfL -X GET  https://api.github.com/repos/mongodb/atlas-sdk-go/releases/latest | jq -r '.tag_name')" >> "$GITHUB_OUTPUT"
       - run:  make update-atlas-sdk
       - name: Verify Changed files
-        uses: tj-actions/verify-changed-files@11ea2b36f98609331b8dc9c5ad9071ee317c6d28
+        uses: tj-actions/verify-changed-files@v20
```

Comment:
can you please use the sha? or you can leave it until tuesday and dependabot will create a PR

Reply:
can I merge like this so snyk understand we're using a version > 17?

Reply:
there's a request from security to keep 3rd party Gh actions (not mantained by GH) using shas instead of pinned version

Reply:
I see, i'll close this then! as it would come up again after we fix the issue.